exporter/dynatraceexporter/config/config.go

```diff
@@ -0,0 +1,58 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"errors"
+	"strings"
+
+	"go.opentelemetry.io/collector/config/configmodels"
+)
+
+// Config defines configuration for the Dynatrace exporter.
+type Config struct {
+	configmodels.ExporterSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
+
+	// URL of the Dynatrace metrics ingest endpoint. Defaults to http://127.0.0.1:14499/metrics/ingest
+	Endpoint string `mapstructure:"endpoint"`
```

Comment:
Do you want to use the confighttp.HttpClientSettigs from core?

Reply:
I wasn't aware it existed. I just looked at another exporter and did what it did. I think it was datadog? I can change to use that if you would prefer?